program/Cargo.toml

```
@@ -0,0 +1,24 @@
+[package]
+name = "eclipse-onchain-program"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+solana-program = "1.9.5"
+thiserror = "1.0.24"
+arrayref = "0.3.6"
+solana-frozen-abi = "1.9.4"
```

```suggestion
solana-frozen-abi = "1.9.5"
```
Should all Solana deps match?